runtime/autoload/man.vim

```diff
@@ -12,6 +12,11 @@ catch /E145:/
   " Ignore the error in restricted mode
 endtry
 
+try
+  call system('env MANPAGER=cat man -l man')
```

Comment:
Do not use `system(string)` if you can: `system(['env', 'MANPAGER=cat', 'man', '-l', 'man'])`.

---

I (Gentoo linux) have exit code 1 on `man -l man`.